src/Sarif.Multitool.FunctionalTests/TestData/StepValuesMustFormOneBasedSequence/InvalidStepValues.sarif

```diff
@@ -1,6 +1,5 @@
-{
-  "$schema": "http://json.schemastore.org/sarif-1.0.0",
-  "version": "1.0.0",
+{  "$schema": "http://json.schemastore.org/sarif-2.0.0",
```

Comment:
>$schema [](start = 4, length = 7)

One hardly knows what to say about diffs this large. You're probably better off looking at "right side only" and treating all the test files as new.